cmd/main.go

```diff
@@ -68,6 +83,15 @@ func init() {
 	rootCmd.Flags().Int64Var(&maxVolumesPerNode, "maxvolumespernode", 0, "limit of volumes per node")
 	rootCmd.Flags().StringVar(&shareRelistInterval, "share-relist-interval", "",
 		"the time between controller relist on the share resource expressed with golang time.Duration syntax(default=10m")
+	rootCmd.Flags().BoolVar(&readOnly, "readonly", false, "does not watch for resource updates")
+}
+
+func loadKubeConfig() (*kubernetes.Clientset, error) {
+	kubeRestConfig, err := client.GetConfig()
+	if err != nil {
+		return nil, err
+	}
+	return kubernetes.NewForConfig(kubeRestConfig)
 }
```

Comment:
Drop this function in favor of just using `client.GetConfig` directly.

Reply:
Please consider previous [comment](https://github.com/openshift/csi-driver-shared-resource/pull/46#discussion_r694669253).